dear_bindings.py

```diff
@@ -425,7 +427,7 @@ def convert_header(src_file, dest_file_no_ext, template_dir):
                         help='Path to output files (generally cimgui). This should have no extension, '
                              'as <output>.h, <output>.cpp and <output>.json will be written.')
     parser.add_argument('-t', '--templatedir',
-                        default="./src/templates",
+                        default=default_template_dir,
                         help='Path to the implementation template directory (default: ./src/templates)')
```

Comment:
I left the `"./src/templates`" in the help string alone.
Let me know if you like it changed as well.